src/features/assets/components/__test__/EditAsset.test.tsx

```diff
@@ -13,6 +13,7 @@ import { fetchAssetsCategory, fetchAssetsInfo, fetchAssetsModel } from './mockAp
 import { mswServer } from 'mocks/mswServer';
 import { ConfirmProvider } from 'material-ui-confirm';
 import { SnackbarProvider } from 'notistack';
+jest.setTimeout(10000);
```

Comment:
Trochę mnie martwi, że test potrzebuje wiecej niż 5s, bo to znaczy, że coś mocno nie bangla jak powinno xD
Ale nie przejmujmy się :D